model/VocabularyConfig.php

```diff
@@ -119,17 +119,28 @@ public function getDataURLs()
                 $mimetypelit = $url->getLiteral('dc11:format');
             }
 
-            // if still not found, guess MIME type using file extension
             if ($mimetypelit !== null) {
-                $mimetype = $mimetypelit->getValue();
+                if ($lang !== null) {
+                    if ($lang == $mimetypelit->getLang()) {
```

Comment:
Wait, is this taking the language from the MIME type literal? That's not how `dc:format` values normally look like...

Reply:
Thanks! Fixed the handling language parameter. (It was never used since I had the secondary preg_match handling the marc data dump according to its filename extension.) I removed the regex as unnecessary.